var/spack/repos/builtin/packages/ocaml/package.py

```diff
@@ -23,6 +23,7 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+import string
```

Comment:
@goxberry You need to remove this unneeded import to have `flake8` tests passing